CineCompass/Controller/MovieDetailsViewController.swift

```diff
@@ -16,19 +17,10 @@ class MovieDetailsViewController: UIViewController {
     @IBOutlet weak var movieOverview: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        movieTitle.text = movieDetails?.title
+        movieReleaseDate.text = movieDetails?.releaseDate
+        movieVoteAvg.text = String(format: "%.1f", movieDetails?.voteAverage ?? -9999.9999)
+        movieGenres.text = (movieDetails?.genres[0].name ?? " ") + "   " + (movieDetails?.genres[1].name ?? " ") + "   " + (movieDetails?.genres[2].name ?? " ")
```

Comment:
you can use "\n" for new line instead of "  "

Reply:
Tek satır olacağını düşündüm, dediğin gibi yapsam bile burda 3 genre ya sahip Bi film değilse fatal error alıyorum, tüm filmlerde olmadı ama karşılaştım 1 2 kere